deploy/gk-deploy

```diff
@@ -129,7 +129,7 @@ eval_output() {
   )
 }
 abort() {
-  ${CLI} delete all,service,deployment,secret --selector="heketi"
+  ${CLI} delete service,deployment,pod --selector="heketi"
```

Comment:
Isn't that a semantic change: before we deleted a secret labeled heketi, now we're deleting a pod labeled heketi.

And why is "all" removed here?